README.md

```diff
@@ -101,6 +102,27 @@ pytest tests/test_client.py --cov=client -v
    - Verify your .env file exists and has correct credentials
    - Check IOL API status
    - Ensure your IOL account is active
+   
+3. If running tests fails with `ModuleNotFoundError: No module named 'httpx'`:
+   - Make sure you're inside the virtual environment:
+     ```bash
+     uv venv
+     source .venv/bin/activate
+     ```
+   - Be sure to run:
+     ```bash
+     uv sync
+     ```
```

Comment:
`uv sync` is part of the setup and running with `uv run` already runs in the virtual environment so this is not needed.